include/natalie/dsl.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "symbol_value.hpp"
+#include "string.hpp"
+#include "value_ptr.hpp"
+#include "global_env.hpp"
+#include "env.hpp"
+
+namespace Natalie {
+
+ValuePtr find_constant(Env *env, Vector<String> paths) 
```

Comment:
I think we could use an `initializer_list` for this, like we do [here](https://github.com/seven1m/natalie/blob/d01faf8d593fda99592087fa431b18570c125024/src/encoding_value.cpp#L16), and get rid of the macro?

Reply:
Good call :) I don't really care about the amount of typing to be honest, as long as it is something clear and straightforward.

Reply:
Addressed this comment and added just a lil small operator to quickly convert strings to symbols, what do you think?

Reply:
I'm not sure! Haha. On the one hand, I like how quick it is to type. On the other, not being a fantastic C++ programmer nor having a lot of experience reading other C++ projects, will this make it harder for new people to understand what's going on?

Going to ponder this one for a bit...

Reply:
As a c++ noob I felt that was intuitive enough when I saw it in serenity, enough that I didn't even question it initially, only to then ask myself "hey, how does this voodoo trick even work???" And find out it's just an operator :) I wouldn't use it extensively but for concepts that are so close like symbols and strings, it kind of makes sense I think :)

Reply:
OK that makes sense. Let's do it.